### PR TITLE
Address review comments and add installing instructions

### PR DESCRIPTION
**Pull request by luomiao**
1. Provide pre-built kops/nodeup binaries and protokube image at:
https://storage.googleapis.com/kops-for-vsphere-cna-storage
2. Update vsphere-dev.md with instructions for installing only option
3. Update the dns-controller image source to cnastorage/dns-controller

And a few minor updates.

### CODE REVIEW

**Comment by msterin** [comment]
https://storage.googleapis.com/kops-for-vsphere-cna-storage returns **AccessDeniedAccess denied.Anonymous users does not have storage.objects.list access to bucket kops-for-vsphere-cna-storage.**. Can we get is on something open for anonymous access , or enable the anonymous access on the current storage ?

**Comment by msterin** [comment]
is there any reason this is separate from #41 ? Not an issue, just curious.

**Comment by luomiao** [comment]
@msterin 
For the AccessDeniedAccess issue: I just added permission to this bucket but seems directly accessing it will result in XML format return. We can try out other options. For now you have to access the files with the whole URLs like: https://storage.googleapis.com/kops-for-vsphere-cna-storage/protokube.tar.gz

"is there any reason this is separate from #41 ? ": It's not separated. This is a new patch need your reviews to be added into vsphere-devleop. After this one is merged, these changes will be included into #41 automatically.

File: docs/development/vsphere-dev.md [hunk 1]
```diff
@@ -73,21 +73,43 @@ vSphere cloud provider support in kops is a work in progress. To try out deployi
 ### Pre-requisites
 + vSphere with at least one ESX, having sufficient free disk space on attached datastore. ESX VM's should have internet connectivity.
 + Setup DNS following steps given in relevant Section above.
-+ Create the VM using this template (TBD).
++ Upload VM template. Steps:
+1. Login to vSphere Client.
+2. Right-Click on ESX host on which you want to deploy the template.
+3. Select Deploy OVF template.
+4. Copy and paste URL for [OVA](TBD).
```
Comment (abrarshivani) [inline docs/development/vsphere-dev.md]: Why OVA is not uploaded?
- If uploaded add timestamp when OVA is uploaded

Comment (luomiao) [reply docs/development/vsphere-dev.md]: Was waiting for @prashima to upload it.
I will try to upload a version to google storage.

Comment (prashima) [reply docs/development/vsphere-dev.md]: On it. Will do today. Sorry about the delay.

**Pull request merged**